implementations/rust/ockam/ockam/src/credential.rs

```diff
@@ -13,7 +13,9 @@ use ockam_core::compat::collections::BTreeMap;
 use ockam_core::errcode::{Kind, Origin};
 use ockam_core::vault::{Signature, SignatureVec};
```

Comment:
I think Verifier can be imported here as well

Reply:
Too much automation.